internal/api/routes_heresphere.go

```diff
@@ -0,0 +1,1402 @@
+package api
```

Comment:
Can we move this into a `heresphere` package and split this up into separate files as needed.

Reply:
The main problem with this is that many heresphere functions rely on functions in the api package.
Seperating it into a heresphere package would create an import cycle because the server.go always needs a reference to the routes from heresphere, and heresphere always needs a reference to GetBaseURL, context_keys, HandleFloat64Value, SceneMarkerCreateInput, SceneFinder, CaptionFinder, ResolverRoot

Reply:
This indicates a smell in the design. I'd suggest taking a look at the DLNA code, since it is doing something somewhat similar. As far as I can determine, there's no reason the heresphere code _needs_ to use the graphql-layer interfaces - you should be able to use the lower level functions to do what you need.